dev/general/config.go

```diff
@@ -310,50 +332,86 @@ func (o *container_v1) Normalize() Config {
-	ProxyServer           string       `xml:"secure-proxy-server,omitempty"`
-	ProxyPort             int          `xml:"secure-proxy-port,omitempty"`
-	ProxyUser             string       `xml:"secure-proxy-user,omitempty"`
-	ProxyPassword         string       `xml:"secure-proxy-password,omitempty"`
-	Dns                   *deviceDns   `xml:"dns-setting"`
-	Ntp                   *deviceNtp   `xml:"ntp-servers"`
-	AckLoginBanner        *util.RawXml `xml:"ack-login-banner"`
-	AuthenticationProfile *util.RawXml `xml:"authentication-profile"`
-	CertificateProfile    *util.RawXml `xml:"certificate-profile"`
-	DomainLookupUrl       *util.RawXml `xml:"domain-lookup-url"`
-	FqdnForceRefreshTime  *util.RawXml `xml:"fqdn-forcerefresh-time"`
-	FqdnRefreshTime       *util.RawXml `xml:"fqdn-refresh-time"`
-	GeoLocation           *util.RawXml `xml:"geo-location"`
-	HsmSettings           *util.RawXml `xml:"hsm-settings"`
-	IpAddressLookupUrl    *util.RawXml `xml:"ip-address-lookup-url"`
-	Ipv6Address           *util.RawXml `xml:"ipv6-address"`
-	Ipv6DefaultGateway    *util.RawXml `xml:"ipv6-default-gateway"`
-	Locale                *util.RawXml `xml:"locale"`
-	LogExportSchedule     *util.RawXml `xml:"log-export-schedule"`
-	LogLink               *util.RawXml `xml:"log-link"`
-	MotdAndBanner         *util.RawXml `xml:"motd-and-banner"`
-	Mtu                   *util.RawXml `xml:"mtu"`
-	PermittedIp           *util.RawXml `xml:"permitted-ip"`
-	Route                 *util.RawXml `xml:"route"`
-	Service               *util.RawXml `xml:"service"`
-	SnmpSetting           *util.RawXml `xml:"snmp-setting"`
-	SpeedDuplex           *util.RawXml `xml:"speed-duplex"`
-	SslTlsServiceProfile  *util.RawXml `xml:"ssl-tls-service-profile"`
-	SyslogCertificate     *util.RawXml `xml:"syslog-certificate"`
-	Type                  *util.RawXml `xml:"type"`
-	UpdateSchedule        *util.RawXml `xml:"update-schedule"`
+type entry_v1 struct {
+	XMLName               xml.Name        `xml:"system"`
+	Hostname              string          `xml:"hostname"`
+	IpAddress             string          `xml:"ip-address,omitempty"`
+	Netmask               string          `xml:"netmask,omitempty"`
+	Gateway               string          `xml:"default-gateway,omitempty"`
+	Timezone              string          `xml:"timezone,omitempty"`
+	Domain                string          `xml:"domain,omitempty"`
+	UpdateServer          string          `xml:"update-server,omitempty"`
+	VerifyUpdateServer    string          `xml:"server-verification"`
+	LoginBanner           string          `xml:"login-banner,omitempty"`
+	ProxyServer           string          `xml:"secure-proxy-server,omitempty"`
+	ProxyPort             int             `xml:"secure-proxy-port,omitempty"`
+	ProxyUser             string          `xml:"secure-proxy-user,omitempty"`
+	ProxyPassword         string          `xml:"secure-proxy-password,omitempty"`
+	Panorama              *devicePanorama `xml:"panorama"`
```

Comment:
I get what you're trying to do here, but I suspect that this wouldn't actually work run against a real PAN-OS with the old and new Panorama server locations...?